packages/dev/loaders/src/SPLAT/splatFileLoader.plugin.ts

```diff
@@ -0,0 +1,103 @@
+import type { ISceneLoaderPluginAsync, ISceneLoaderPluginFactory, ISceneLoaderPlugin, ISceneLoaderAsyncResult, ISceneLoaderProgressEvent } from "core/Loading/sceneLoader";
+import { SPLATFileLoaderMetadata } from "./splatFileLoaderMetadata";
+import { GaussianSplattingMesh } from "core/Meshes/GaussianSplatting/gaussianSplattingMesh";
+import type { AssetContainer } from "core/assetContainer";
+import type { Scene } from "core/scene";
+
+declare module "core/Loading/sceneLoader" {
+    // eslint-disable-next-line jsdoc/require-jsdoc
+    export interface SceneLoaderPluginOptions {
+        /**
+         * Defines options for the splat loader.
+         */
+        [SPLATFileLoaderMetadata.Name]: {};
+    }
+}
+
+/**
+ * @experimental
+ * SPLAT file type loader.
+ * This is a babylon scene loader plugin.
+ */
+export class SPLATFileLoader implements ISceneLoaderPluginAsync, ISceneLoaderPluginFactory {
+    /**
+     * Defines the name of the plugin.
+     */
+    public readonly name = SPLATFileLoaderMetadata.Name;
+
+    /**
+     * Defines the extensions the splat loader is able to load.
+     * force data to come in as an ArrayBuffer
+     */
+    public readonly extensions = SPLATFileLoaderMetadata.Extensions;
+
+    //private _loadingOptions: SPLATLoadingOptions;

```

Comment:
Is this supposed to be commented out?

Reply:
I didn't make any changes to this code, it's just moved from the .ts file to the .plugin.ts.

Reply:
Yup, I see that. Maybe remove anyways since it's completely unreferenced.

Reply:
@CedricGuillemet will you fix this in your splat loader PR?
https://github.com/BabylonJS/Babylon.js/pull/15501

Reply:
yes!